docs/src/rust-client/ephemeral_note_how_to.md

```diff
@@ -1,12 +1,12 @@
-# How to Use Ephemeral Notes 
+# How to Use Ephemeral Notes
 
-*Using ephemeral notes for optimistic note consumption*
+_Using ephemeral notes for optimistic note consumption_
```

Comment:
I think we call those notes now `unauthenticated notes`. Can you change that everywhere?

`Unauthenticated notes` are being `consumed optimistically`.